TESTOrderScript.js

```diff
@@ -24,11 +31,15 @@ function addRow() {
             $(`[rel='js-quantity__bi-column--left--${rowNumber}']`).append(`<p class="order-form__servings--text order-form__servings--text${rowNumber}" rel="js-order-form__servings${rowNumber}"></p>`);
             $(`[rel='js-order-form__quad-column2--${rowNumber}']`).append(`<div class="quantity__bi-column quantity__bi-column--right" rel="js-quantity__bi-column--right--${rowNumber}"></div>`);
             $(`[rel='js-quantity__bi-column--right--${rowNumber}']`).append(`<select type="text" name="quantity" id="quantity${rowNumber}" onclick="findRowNumber(this);calculatePrice(this)"  class="order-form__input order-form__quantity order-form__quantity${rowNumber} entry${rowNumber}" rel="js-order-form__quantity${rowNumber}"></select>`);
-            $(`[rel='js-order-form__quantity${rowNumber}']`).append('<option value="1" selected="selected">1 batch</option>');
-            $(`[rel='js-order-form__quantity${rowNumber}']`).append('<option value="2">2 batches</option>');
-            $(`[rel='js-order-form__quantity${rowNumber}']`).append('<option value="3">3 batches</option>');
-            $(`[rel='js-order-form__quantity${rowNumber}']`).append('<option value="4">4 batches</option>');
-            $(`[rel='js-order-form__quantity${rowNumber}']`).append('<option value="5">5 batches</option>');
+
+            // TODO: for example, what do you think of this?
```

Comment:
this works if written like:
const leftMainBiColumn = `[rel='js-order-form__bi-column__left--${rowNumber}']`;

 $(leftMainBiColumn).append(`<div class="order-form__quad-......

The function has been updated to use variables for finding the elements to append to.

Reply:
It's probably fine either way. I guess for consistency sake you could stick to the same pattern, but no one should really think twice about it either way. The only advantage of extracting to a variable is it keeps the line lengths down, which lots of linters will yell about too long of lines.

Reply:
Opted for using all variables to it all looks and reads in a consistent manner.